curryreader.py

```diff
@@ -85,7 +85,7 @@ def read(inputfilename='', plotdata = 1, verbosity = 2):
     filename = os.path.basename(filepath)
 
     try:
-        basename, extension = filepath.split(".", maxsplit=1)
+        basename, extension = ".".join(filepath.split(".")[:-1]), filepath.split(".")[-1]  # BUG (cannot handle dots in paths) FIXED

```

Comment:
seems like [rsplit](https://docs.python.org/3/library/stdtypes.html#str.rsplit) should work, unless I'm missing something?
```suggestion
        basename, extension = filepath.rsplit(".", maxsplit=1)
```

Reply:
til :D